src/pages/en/core-concepts/framework-components.md

```diff
@@ -56,9 +56,15 @@ import MyReactComponent from '../components/MyReactComponent.jsx';
 </html>
 ```
 
+>💡 _Remember: all imports must live at the **top** of your Astro component script!_
+
+By default, your framework components will render as static HTML. This is useful for templating components that are not interactive and avoids sending any unnecessary JavaScript to the client.
+
 ## Hydrating Interactive Components
 
-A framework component can be hydrated using a `client:*` directive. This is a component attribute to define how your component should be **rendered** and **hydrated**. It describes whether your component should be rendered at build-time, and when your component's JavaScript should be loaded by the browser, client-side.
+A framework component can be made interactive (hydrated) using a `client:*` directive. This is a component attribute to define how your component should be **rendered** and **hydrated**.
```

Comment:
Nit? This might help underline `client:*` itself is not a valid client directive.

```suggestion
A framework component can be made interactive (hydrated) using one of the `client:*` directives. This is a component attribute to define how your component should be **rendered** and **hydrated**.
```

Reply:
(otherwise this all looks great!)